apps/web/api/api-classes/auth.ts

```diff
@@ -0,0 +1,55 @@
+import type { AxiosInstance } from "axios";
+import {
+	AccessTokenCookie,
+	RefreshTokenCookie,
+} from "../config/cookies.client";
+
+export class Auth {
+	constructor(private http: AxiosInstance) {}
+
```

Comment:
Rename to `AuthAPI` for easier auto-completion and intellisense. General names like `Auth` may conflict with other 3rd-party or built-in classes and functions.

Reply:
I will do it, keep this for me.